deployments/fpga_admissionwebhook/mappings-collection.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: fpga.intel.com/v1
+kind: AcceleratorFunction
+metadata:
+  name: arria10-nlb0
+spec:
+  interfaceId: ce48969398f05f33946d560708be108a
+  afuId: d8424dc4a4a3c413f89e433683f9040b
+---
+apiVersion: fpga.intel.com/v1
+kind: AcceleratorFunction
+metadata:
+  name: arria10-nlb3
+spec:
+  interfaceId: ce48969398f05f33946d560708be108a
```

Comment:
Should we be able here to re-use symbolic name of FpgaRegion object ?

Reply:
Good question! It seems this field is redundant (and is unused in the code) because the symbolic name of FpgaRegion is already a part of AF's name.